docs/getting_support.rst

```diff
@@ -3,45 +3,435 @@
+
+.. _Logging In:
+
+Logging In
+----------
+
+Once your account has been activated, go to the `Support Page`_ and log
+in with your new account.
+
+.. _Support Page: https://support.freedom.press
+
+|Login|
+
+After logging in, you will be be taken to the Home page. In the top
+menu bar, you will see helpful links to a variety of resources. The most
+important links are:
+
+* **Projects**: The list of projects you have access to.
+* **My Page**: An overview of the open issues that were either created
+  by you or are assigned to you.
+
+|Home|
+
+Click on **Projects** in the top menu bar. You'll see a project for your
+SecureDrop instance listed underneath the parent SecureDrop project.
+Only Freedom of the Press Foundation staff and people within your
+organization have access to your instance's project.
+
+|Projects|
+
+Click on the project link. You'll be shown an **Overview** of the project and
+all of its issues.
+
+|Overview|
+
+To view the list of open issues, click the **Issues** button to open the issues
+pane.
+
+|Issues|
+
+.. |Login| image:: images/support/login.png
+.. |Home| image:: images/support/home.png
+.. |Projects| image:: images/support/projects.png
+.. |Overview| image:: images/support/overview.png
+.. |Issues| image:: images/support/issues.png
+
+
+Onboarding Issue
+----------------
+
+By the time you log in, we'll have created an initial issue for
+onboarding, called "Onboarding Verification". Please write a reply on
+this issue, which will let us know that you succeeded in logging in and
+finding your project page.
+
+|OnboardingIssue|
+
+To write a reply, click **Edit** button at the top of the issue.
+Write your reply in the Notes text box, then click **Submit**.
+
+|EditIssue1|
+
+We'll reply to the issue and start a little exchange to explain some
+features of the platform and answer any questions you might have. The
+goal here is to make sure you're comfortable and confident using the new
+Support site.
+
+|EditIssue2|
+
+Once we're satisfied you're ready to use the new site, we'll close the
+issue.
+
+Once we're finished with the onboarding issue, you're ready to use the Support
+site. If you want to additionally set up encrypted email notifications, head
+over to the :ref:`Encrypted Email Overview <encrypted_email_overview>`.
+
+.. |OnboardingIssue| image:: images/support/onboarding_issue.png
+.. |EditIssue1| image:: images/support/edit_issue_1.png
+.. |EditIssue2| image:: images/support/edit_issue_2.png
+
+Creating a new issue
+--------------------
+
+To create a new issue, click **New issue** and fill out the Subject and
+Description fields. All other fields can be left blank, although you can
+assign a priority if you want. When we see the issue, we will respond
+and assign it to the appropriate person. If we're waiting for a reply
+from someone in your organization, we may assign it back to you.
+
+|NewIssue|
+
+.. |NewIssue| image:: images/support/new_issue.png
+
+Enabling two-factor authentication
```

Comment:
I would suggest we make this part of the "creating an account" portion, prior to them testing out ticket submission and such -- this helps establish that 2FA is part of the account setup process, and not something to maybe pick up later.